docs/Installation-Guide.rst

```diff
@@ -94,7 +94,8 @@ The ``.exe`` and ``.dll`` files will be in ``LightGBM/`` folder.
 
 **Note**: You may need to run the ``cmake -G "MinGW Makefiles" ..`` one more time if you encounter the ``sh.exe was found in your PATH`` error.
 
-It is recommended to use **Visual Studio** for its better multithreading efficiency in **Windows** for many-core systems (see `FAQ <./FAQ.rst#lightgbm>`__ Question 4 and Question 8).
+It is recommended to use **Visual Studio** for its better multithreading efficiency in **Windows** for many-core systems
+(see `FAQ <./FAQ.rst#general-lightgbm-questions>`__ `Question 4 <./FAQ.rst#i-am-using-windows-should-i-use-visual-studio-or-mingw-for-compiling-lightgbm>`__ and `Question 8 <./FAQ.rst#cpu-usage-is-low-like-10-in-windows-when-using-lightgbm-on-very-large-datasets-with-many-core-systems>`__).
```

Comment:
@hayesall Please remove the link from word `FAQ`. It's very nonintuitive to have two hyperlinks followed one by one, because you cannot visually distinguish where the first link is ended and the second one starts. So, one user can click on `FAQ` and will be force to scroll down for `Question 4` while another user click on `Question 4` and get the answer instantly.


Also, please update the link in Python wrapper's README in the same manner.

> Also, it'll cool if you upgrade one another remaining link to FAQ so that it'll point to individual entry.
    https://github.com/microsoft/LightGBM/blob/7da11ffeacf79be81839431b1cbc7c293ba7d9bd/python-package/README.rst#build-with-mingw-w64-on-windows
    ![image](https://user-images.githubusercontent.com/25141164/62136388-ae6e4900-b2ec-11e9-8f9d-6fba1f3af5ef.png)
   https://github.com/microsoft/LightGBM/pull/2293#pullrequestreview-268301671

Reply:
This should be resolved with the latest commit. I ran `grep -r "FAQ" *` and didn't find anywhere else that needed patched.